pyro/contrib/glmm/glmm.py

```diff
@@ -220,7 +220,7 @@ def bayesian_linear_model(design, w_means={}, w_sqrtlambdas={}, re_group_sizes={
             alpha, beta = re_alphas[name], re_betas[name]
             group_p = alpha.shape[-1]
             G_prior = dist.Gamma(alpha.expand(tau_shape + (group_p,)),
-                                 beta.expand(tau_shape + (group_p,)))
+                                 beta.expand(tau_shape + (group_p,))).independent(1)
```

Comment:
@eb8680 @ae-foster this appears to be a bug caught by the new `@broadcast`-by-default. Can you confirm that this is correct?

Reply:
Yep you are right @fritzo - the `group_p` dimension should be under `independent(1)`

Reply:
Nice catch :)